use-shopping-cart/utilities/serverless.js

```diff
@@ -24,7 +24,8 @@ function validateCartItems(inventorySrc, cartDetails) {
       quantity: cartDetails[id].quantity
     }
 
-    if (typeof cartDetails[id].product_data?.metadata === 'object') {
```

Comment:
Just wondering, what was wrong with the optional chaining?

Reply:
I ran into an issue with Netlify because of it. The solution would have been to set the environment to be node v14, Netlify's node version wasn't compatible with it. I personally wanted to downgrade this bit just so that users in the future won't get caught up in it, hope that's okay with you!

Reply:
Yeah, you're good, just wanted to know. I'll make sure that if I'm working in serverless stuff to avoid brand new Node features.